### PR TITLE
docs: fix bugs in React render engine documentation

### PR DESCRIPTION
**Pull request by Arya-Gupta**
Issue: https://github.com/asyncapi/generator/issues/910
Signed-off-by: Arya Gupta <arya.gupta99999@gmail.com>

### CODE REVIEW

File: docs/react-render-engine.md [hunk 1]
```diff
@@ -33,23 +33,23 @@ export default function({ asyncapi, params, originalAsyncAPI }) {
...
-For further information about components, props etc. see the [Generator React SDK](https://github.com/asyncapi/generator-react-sdk)
+For further information about components, props etc, see the [Generator React SDK](https://github.com/asyncapi/generator-react-sdk)
 
 ### Common assumptions
 
 1. Generator renders all files located in the `template` directory if they meet the following conditions:
     - `File` is the root component
     - The file is not in the list of `nonRenderableFiles` in the template configuration
 1. New lines are automatically added after each `Text` component.
-1. The props you have access to in rendering function is:
```
Comment (pratik2315) [inline docs/react-render-engine.md:44]: The props you have access to in the rendering function are:

File: docs/react-render-engine.md [hunk 2]
```diff
@@ -33,23 +33,23 @@ export default function({ asyncapi, params, originalAsyncAPI }) {
...
-   - `asyncapi` that is a parsed spec file object. Read the [API](https://github.com/asyncapi/parser-js/blob/master/API.md#AsyncAPIDocument) of the Parser to understand what structure you have access to in this parameter.
-   - `originalAsyncAPI` that is an original spec file before it is parsed. 
+1. The props you have access to in rendering function are:
```
Comment (pratik2315) [inline docs/react-render-engine.md:44]: The props you have access to in the rendering function are:

Comment (pratik2315) [reply docs/react-render-engine.md:44]: Hey @Arya-Gupta i've added 'in the' in line 44

Comment (pratik2315) [reply docs/react-render-engine.md:44]: Just some minor stuff haha

Comment (Arya-Gupta) [reply docs/react-render-engine.md:44]: Oh, I just realised! Sure, np.

File: docs/react-render-engine.md [hunk 3]
```diff
@@ -33,23 +33,23 @@ export default function({ asyncapi, params, originalAsyncAPI }) {
 
 The exported default function returns a `File` component as a root component which the generator uses to determine what file should be generated. In our case, we overwrite the default functionality of saving the file as `MyTemplate.js` but instead use the filename `asyncapi.md`. It is then specified that we should render `Some text that should render as is\n` within that file. Notice the `\n` character at the end, which is automatically added after the `Text` component. 
 
-For further information about components, props etc. see the [Generator React SDK](https://github.com/asyncapi/generator-react-sdk)
+For further information about components, props etc, see the [Generator React SDK](https://github.com/asyncapi/generator-react-sdk)
```
Comment (Florence-Njeri) [inline docs/react-render-engine.md:36]:
```suggestion
For further information about components, props, etc, see the [Generator React SDK](https://github.com/asyncapi/generator-react-sdk)
```

**Review by pratik2315 — APPROVED** [review]

**Pull request merged**